src/docs/components/nav/mobile-nav.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
```

Comment:
Would love to have the nav come from below as we discussed, but it can come on a future interaction 😄

Reply:
Let me see what I can whip up, might as well take care of it while it's fresh!